janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java

```diff
@@ -137,6 +137,17 @@
             " If this configuration option is not provided but graph.graphname is, the table will be set" +
             " to that value.",
             ConfigOption.Type.LOCAL, "janusgraph");
+    
+    public static final ConfigOption<String> HBASE_SNAPSHOT =
+            new ConfigOption<>(HBASE_NS, "snapshot-name",
+            "The name of an exising HBase snapshot to be used by HBaseSnapshotInputFormat",
+            ConfigOption.Type.LOCAL, "janusgraph");
+    
+    public static final ConfigOption<String> HBASE_SNAPSHOT_RESTORE_DIR =
+            new ConfigOption<>(HBASE_NS, "snapshot-restore-dir",
+            "The tempoary directory to be used by HBaseSnapshotInputFormat to restore a snapshot." +
+            " This directory should be on the same File System as the HBase root dir.",
+            ConfigOption.Type.LOCAL, "/tmp");
```

Comment:
I wonder what the best way is.  The directory can be on hdfs normally.

Reply:
Is there a way to get a temporary HDFS directory if it's available or otherwise fall back on `java.io.tmpdir`? Either way not sure how you could express that as the default value instead of `"/tmp"` above. Maybe you could just omit the default and initialize it as appropriate in the constructor and then document the behavior as part of the parameter description string.